ArcadiaParties-SPA/src/app/app.module.ts

```diff
@@ -18,7 +18,11 @@ import { UserModule } from './user/user.module';
 import { TitleEffect } from './effects/effect';
 import { DepartmentModule } from './department/department.module';
 import { TokenInterceptorProvider } from './services/token-interceptor';
-import { UnauthorizedInterceptorProvider } from './services/unauthorized-interceptor';
+import { AppSettingsService } from './app-settings/app-settings.service';
+
+export function initializeAppSettings(appSettingsService: AppSettingsService) {
```

Comment:
It doesn't need to be exported.

Reply:
Also it can be inlined, because it is quite simple.

Reply:
Inlined.